src/client/Components/Util/LogTable.jsx

```diff
@@ -1,12 +1,25 @@
+// @flow
+
 import React from 'react';
 
-export default class LogTable extends React.Component {
-  constructor(props) {
+type Props = {
+  github_repo: ?string
+};
```

Comment:
`LogTableProps`

Reply:
Same argument applies as to why I don't have it for `State` variables. Whatever we decide for that we'll propogate.